sysutils/bash-completion/files/port

```diff
@@ -30,7 +30,7 @@ _port()
                         mode=cmdfile
                         cmdfile=${words[((count+1))]}
                         ;;
-                    activate|archive|archivefetch|build|cat|cd|checksum|clean|configure|\
+                        activate|archive|archivefetch|build|bump|cat|cd|checksum|clean|configure|\
```

Comment:
```suggestion
                    activate|archive|archivefetch|build|bump|cat|cd|checksum|clean|configure|\
```

Please keep indentation aligned with the other options.

Reply:
Done